closed/src/java.base/share/native/libjncrypto/NativeCrypto.c

```diff
@@ -22,6 +22,20 @@
  * ===========================================================================
  */
 
+#if defined(_AIX)
+#include <dlfcn.h>
+#include <sys/ldr.h>
+#define DLFCN_LDINFO_SIZE (sizeof(struct ld_info) + _XOPEN_PATH_MAX + _XOPEN_NAME_MAX)
+#elif defined(__APPLE__)
```

Comment:
Each `#elif`, `#else` and `#endif` directive should be followed by a comment with the relevant previous condition.
```c
#if defined(_AIX)
...
#elif defined(__APPLE__) /* defined(_AIX) */
...
#elif defined(__linux__) /* defined(__APPLE__) */
...
#elif defined(_WIN32) /* defined(__linux__) */
...
#endif /* defined(_AIX) */
```
Please apply throughout.

Reply:
Added comments throughout.